app/models/download.rb

```diff
@@ -115,7 +116,11 @@ def page_title
   end
 
   def chapters
-    work.chapters.order('position ASC').where(posted: true)
+    if @include_draft_chapters
+      work.chapters.order("position ASC")
+    else
+      work.chapters.order("position ASC").where(posted: true)
```

Comment:
You can use the posted scope in some places:

https://github.com/otwcode/otwarchive/blob/b8f0950323f2b82eaf89ace957d0e739b6ad8051/app/models/chapter.rb#L51

but it's not a blocker.